libs/shared-atomic-design-components/src/lib/molecules/FilterDropdown.tsx

```diff
@@ -66,12 +73,22 @@ const FilterDropdown = ({
       >
         {items.map((item) => (
           <li key={item.value}>
-            <Checkbox
-              handleChange={() => onChange(item.value)}
-              checked={selected.includes(item.value)}
-            >
-              {item.label}
-            </Checkbox>
+            {singleSelect ? (
```

Comment:
@helloanil, couldn't we use a `FormSelect` component here instead of creating a new variant in the `FilterDropdown.tsx`?